### DISCUSSION TITLE
Multi Tenant Azure

### DISCUSSION THREAD

**Discussion by lucamazzasette** [cloud-carbon-footprint/cloud-carbon-footprint#1176] [Q&A]
I need to add into CCF another tenant, I try to add a new :
AZURE_CLIENT_ID=
AZURE_CLIENT_SECRET=
AZURE_TENANT_ID=

but CCF read only last.
I don't find a possible solution into guide but I don't think the insolit request.
Regrds

**Comment by ericksod** [comment]
Hi @lucamazzasette!
We do not currently have multi account support.
To "add another tenant" you'll have to have a separate instance of ccf, or connect to a higher level account that has visibility into both of the tenants.
Hope that makes sense!
